tests/integrationTests/inputs/monitors/radiationFlux.yaml

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
this input looks like one that ablate had already processed?  Do you the the original that used text based anchors?